app/src/main/java/org/mozilla/fenix/collections/CollectionCreationUIView.kt

```diff
@@ -137,7 +137,7 @@ class CollectionCreationUIView(
                 view.bottom_button_bar_layout.setOnClickListener(null)
                 view.bottom_button_bar_layout.isClickable = false
 
-                val drawable = view.context.getDrawable(R.drawable.ic_close)
+                val drawable = ContextCompat.getDrawable(view.context, R.drawable.ic_close)
```

Comment:
`ContextCompat` isn't sufficient for vector compat. Internally it has the following if statement:
```java
if (Build.VERSION.SDK_INT >= 21) {
    return context.getDrawable(id);
} else ...
```

You must use [`AppCompatResources.getDrawable`](https://medium.com/androiddevelopers/using-vector-assets-in-android-apps-4318fd662eb9) instead.